### PR TITLE
Fixing address scramble issue, as well as misc. timing related issues

### PR DESCRIPTION
**Pull request by erbr-ot**
It has been seen for a while, that changing the order of the LL-tests could make quite a number of tests fail now and again.
During debugging of this, I found the below mentioned error in the address scrambling as well as the error in the has_event util function.

Address scramble utils fixed - previous version swapped bytes in
address to create new address, this fails if bytes are identical

has_event util timeout handling was incorrect - waited longer. Now
the timeout is correct. However this then calls for the general update
to timeouts.

Added formatted timing info to trace-out

### CODE REVIEW

File: src/components/basic_commands.py [hunk 1]
```diff
@@ -2828,11 +2830,12 @@ def has_event(transport, idx, to):
...
         if count > 0:
             break;
-        
+
+        to_tmp = 100 - ((transport.last_t - start_t) / 1000);
         to -= 100;
         if to >= 0:
-            transport.wait(100);
-        
+            transport.wait(int(to_tmp));
```
Comment (wopu-ot) [inline src/components/basic_commands.py:2837]: Why is this checking for `to >= 0` when the actual waiting time would be in `to_tmp`?

File: src/components/preambles.py [hunk 2]
```diff
@@ -446,19 +446,19 @@ def preamble_device_address_set(transport, idx, trace):
     Scramble the company_id part of the Bluetooth address.
 """
 def address_scramble_OUI(address):
-    return (address & 0x00ff00ffffff) | ((address << 16) & 0xff0000000000) | ((address >> 16) & 0x0000ff000000);
+    return (address & 0x00ff00ffffff) | (~address & 0xff0000000000) | (~address & 0x0000ff000000);
```
Comment (wopu-ot) [inline src/components/preambles.py:449]: As far as I can tell, this could be written more concisely as `address ^ 0xff00ff000000`; similarly below.

**Review by wopu-ot — CHANGES_REQUESTED** [review]

File: src/components/preambles.py (hunk 2 above)
Comment (erbr-ot) [reply src/components/preambles.py:449]: Agree - will update

File: src/components/basic_commands.py (hunk 1 above)
Comment (erbr-ot) [reply src/components/basic_commands.py:2837]: Well to>=0 check is re. total time down-count, to_tmp is local to each iteration to signify how much of the 100 was not already spent in call transport.recv(idx, 5, 100).

**Review by wopu-ot — APPROVED** [review]

**Pull request merged**